kuma/users/models.py

```diff
@@ -166,6 +166,13 @@ def beta_tester(self):
                 self.user.groups.values_list('name', flat=True))
 
     @property
+    def is_banned(self):
+        return self.user.bans.filter(is_active=True).exists()
+
+    def active_ban(self):
+        if self.is_banned:
+            return self.user.bans.filter(is_active=True)[:1][0]
+
```

Comment:
Nice refactor. We should add a couple corresponding tests to `kuma/users/tests/test_models.py` ... I can help do that if needed.